### ISSUE TITLE
Script Notifications Stopped Working

### ISSUE THREAD

**Issue by mwegner** [Tautulli/Tautulli#640]
Script notifications have stopped working for me.  I'm not sure what the last PlexPy version was when things were okay (and I dug around the history for the notifications files and didn't see anything glaring).  Python isn't my strongest language, so thought I'd report:

All script notifications now error out with the debug log showing "None" as the selected action.  Here's a playback session starting:

```
2016-03-28 21:07:23 - DEBUG :: Thread-17 : PlexPy Monitor :: Session 307 has started.
2016-03-28 21:07:24 - DEBUG :: Thread-22 : PlexPy Notifiers :: Trying to run notify script, action: None, arguments: [u'29026,307']
2016-03-28 21:07:24 - DEBUG :: Thread-22 : PlexPy Notifiers :: No script selected, exiting...
```

That session stopping:

```
2016-03-28 21:11:23 - DEBUG :: Thread-21 : PlexPy Monitor :: Session 307 has stopped.
2016-03-28 21:11:23 - DEBUG :: Thread-21 : PlexPy ActivityProcessor :: Fetching metadata for item ratingKey 29026
2016-03-28 21:11:26 - DEBUG :: Thread-21 : PlexPy Monitor :: Removing sessionKey 307 ratingKey 29026 from session queue
2016-03-28 21:11:27 - DEBUG :: Thread-23 : PlexPy Notifiers :: Trying to run notify script, action: None, arguments: [u'29026,307']
2016-03-28 21:11:27 - DEBUG :: Thread-23 : PlexPy Notifiers :: No script selected, exiting...
```

And here's new content going into the library:

```
2016-03-28 21:30:24 - DEBUG :: Thread-12 : PlexPy Monitor :: Library item 33461 has been added to Plex.
2016-03-28 21:30:24 - DEBUG :: Thread-13 : PlexPy Monitor :: Session 299 has stopped.
2016-03-28 21:30:24 - DEBUG :: Thread-13 : PlexPy ActivityProcessor :: Fetching metadata for item ratingKey 29228
2016-03-28 21:30:24 - DEBUG :: Thread-13 : PlexPy Monitor :: Removing sessionKey 299 ratingKey 29228 from session queue
2016-03-28 21:30:25 - DEBUG :: Thread-25 : PlexPy Notifiers :: Trying to run notify script, action: None, arguments: [u'33461,']
2016-03-28 21:30:25 - DEBUG :: Thread-25 : PlexPy Notifiers :: No script selected, exiting...
2016-03-28 21:30:25 - DEBUG :: Thread-26 : PlexPy Notifiers :: Trying to run notify script, action: None, arguments: [u'29228,299']
2016-03-28 21:30:25 - DEBUG :: Thread-26 : PlexPy Notifiers :: No script selected, exiting...
```

And this version of PlexPy (1.3.13):

```
Git Branch: master
Git Commit Hash:    5cc6e0b172f50007d902302144b04ddbf7197586
Arguments:  ['--nolaunch']
Platform:   Windows 10
Python Version: 2.7.11 (v2.7.11:6d1b6a68f775, Dec 5 2015, 20:32:19) [MSC v.1500 32 bit (Intel)]
```

So it looks like notify_action is being dropped somewhere along the way, but I'm not familiar enough with the PlexPy code base to much debugging on my own :/

**Comment by JonnyWong16** [comment]
Bug in latest version, I think.

Try changing this line: https://github.com/drzoidberg33/plexpy/blob/master/plexpy/notifiers.py#L490
to 

```
scripts.notify(message=body, subject=subject, notify_action=notify_action, **kwargs)
```

**Comment by mwegner** [comment]
Oh yep, that did it!  Thanks.

**Comment by JonnyWong16** [comment]
Fixed in v1.3.14. You might have to undo the change to the code in order to update.

```
git reset --hard HEAD
```
